 2:22:22.28  Test starded without threads
00:22:22.890 Python Module Path 'ScriptKit' added.
00:22:22.890 Python Module Path 'ScriptKit/TroubleMaker' added.
00:22:22.890 Python Module Path 'Scripts/Gateway/SystemAreas/OrderProcessing/PythonModules' added.
00:22:22.890 Python Module Path 'Scripts/Gateway/PythonModules' added.
00:22:23.265 2020-02-19 Root suite 'Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/AlgoExecLimits' filtered execution.
00:22:23.282 Suite '.'. [0 test cases, 1 sub-suites]
00:22:34.797 Suite 'Synthetic'. [0 test cases, 2 sub-suites]
Traceback (most recent call last):
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py", line 33, in invoke_script
    exec(code, globals())
  File "Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/AlgoExecLimits\Synthetic\test_suite_su.py", line 19, in <module>
    generator.Open()
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\IHEGenerator.py", line 44, in Open
    self.__generator.Open(self.__name)
  File "C:\xpit.com\applications\ScriptFramework\ScriptKit\ModuleAgent.py", line 8898, in Open
    return _ModuleAgent.IIHEGenerator_Open(self, name)
RuntimeError: Failed to resolve interface  (from Interface): interface not found.
Available interfaces:
name=\DEVICE\TCPIP_{CFF7A875-51FB-48C1-9B78-ABA35ACC4BDF}, ip=172.26.65.55, network=172.26.65.0, up=1
name=\DEVICE\TCPIP_{1F777394-0B42-11E3-80AD-806E6F6E6963}, ip=127.0.0.1, network=127.0.0.0, up=1


00:22:34.953 Suite 'Synthetic' setup failed, child test cases and sub-suites will be skipped.
00:22:35.703 Root suite 'Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/AlgoExecLimits' execution finished. It took 0:00:12.422298.
00:22:35.703 Test cases results: 7 total, 0 succeeded, 0 failed, 7 skipped.
